core/src/main/java/io/questdb/griffin/TableBackupManager.java

```diff
@@ -0,0 +1,78 @@
+package io.questdb.griffin;
+
+import io.questdb.cairo.AppendMemory;
+import io.questdb.cairo.CairoConfiguration;
+import io.questdb.cairo.CairoEngine;
+import io.questdb.cairo.CairoException;
+import io.questdb.cairo.CairoSecurityContext;
+import io.questdb.cairo.ColumnType;
+import io.questdb.cairo.SymbolMapWriter;
+import io.questdb.cairo.TableReader;
+import io.questdb.cairo.TableReaderMetadata;
+import io.questdb.cairo.TableUtils;
+import io.questdb.cairo.TableWriter;
+import io.questdb.cairo.sql.RecordCursor;
+import io.questdb.std.Files;
+import io.questdb.std.FilesFacade;
+import io.questdb.std.str.Path;
+
+public class TableBackupManager {
```

Comment:
if we use SQL to backup tables, which is likely, this code can move into SQLCompiler

Reply:
Yes, I was think of that, but SQLCompiler is already very long ...
However, I am not bothered either way, it does make sense to put it into SQLCompiler